#!/usr/bin/python3

def raise_exception_msg(message=""):
    """function that Raise exception NameError with a message."""
    raise NameError(message)
